sdf.md

```diff
@@ -2188,6 +2181,34 @@ using the `sdfThing` quality.
 {: sourcecode-name="example-sdfthing-refrigerator-freezer.sdf.json"}
 {: #exa-sdfthing-fridge-freezer}
 
+# Some Changes From Earlier Drafts {#earlier}
+
+[XXX add a pointer to this to new 1.1 intro once PR#158 is merged, as
+in:\\
+Finally, {{earlier}} provides some information that can be useful in
+upgrading earlier, pre-standard models and implementations to SDF base.]
+
+This appendix is informative.
+
+The present document provides the SDF base definition.
+Previous revisions of SDF have been in use for several years, and both
+significant collections of older SDF models and older SDF conversion
+tools are available today.
+This appendix provides a brief checklist that can aid in upgrading
+these to the standard.
+
+* The quality `unit` was previously called `units`.
+* `sdfType` was developed out of a concept previously called `subtype`.
+* `sdfChoice` is the preferred way to represent JSO `enum` (only a
+  limited form of which is retained), and also the way to represent
+  JSO `anyOf`.
+* the length of text strings (as used with `minLength`/`maxLength`
+  constraints) was previously defined in bytes.
+  It now is defined as the number of characters (Unicode scalar
+  values, to be exact): a length in bytes is not meaningful unless
```

Comment:
Is a colon the right choice here, or should it rather be a period/semicolon?

Reply:
Both works, but semicolon probably is more conventional. Can you do a suggestion in your comment?  I can't really edit right now.

Reply:
Sure, here you go :)

```suggestion
  values, to be exact); a length in bytes is not meaningful unless
```